### PR TITLE
feat: Consolidate tips commands under one cog

### PR DESCRIPTION
**Pull request by GabeMillikan**
# Selecting a Tip

https://user-images.githubusercontent.com/44247924/219840624-a0be2b8a-efe1-453a-8505-feb367c24f98.mp4

# Pinging a User

https://user-images.githubusercontent.com/44247924/219840955-d56737d1-13b3-4de5-9eef-04b78e85f620.mp4

# Anonymous Tips

https://user-images.githubusercontent.com/44247924/219842662-9e43ffd2-ea8b-44cd-8aa2-1021b5079d7e.mp4

# Relevant Threads
(private to contributors) https://discord.com/channels/238956364729155585/1040360228388098159/1076232286216921100
(public) https://discord.com/channels/238956364729155585/1047785196063309835/1048956051283791943

### CODE REVIEW

File: cogs/Tips.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+from util.EmbedBuilder import EmbedBuilder
+
+TIPS = {
+    "Ask Your Question": (
+        "You can be helped sooner if you simply ask your question, instead of "
+        "asking if you can ask a question or if anyone is available to help you.\n\n"
+        "Instead of asking "
+        '"Does anyone know X?", "Can someone help me with Y?", or "Are there any experts in Z?" '
+        "just ask your question or describe your problem directly."
+    ),
```
Comment (GabeMillikan) [inline cogs/Tips.py:17]: image

File: cogs/Tips.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        "This message:\n"
+        "**\\`\\`\\`java\n"
+        'System.out.println("Code formatting is neat!");\n'
+        "\\`\\`\\`**\n\n"
+        "Will look like this:\n"
+        "```java\n"
+        'System.out.println("Code formatting is neat!");\n'
+        "```\n"
+        "Go ahead, try it! You can copy/paste the message above."
+    ),
```
Comment (GabeMillikan) [inline cogs/Tips.py:32]: image

File: cogs/Tips.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    "Let Us Know What You've Already Tried": (
+        "Please show and describe the steps you've already completed so that we may assist you further.\n"
+        "This way, we can start helping you right where you left off and can save everyone's time."
+    ),
```
Comment (GabeMillikan) [inline cogs/Tips.py:36]: Replaces `/attempt`
image

File: cogs/Tips.py [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+from util.EmbedBuilder import EmbedBuilder
+
+TIPS = {
+    "Ask Your Question": (
+        "You can be helped sooner if you simply ask your question instead of "
+        "asking if you can ask a question or if anyone is available to help you.\n\n"
+        "Instead of asking "
+        '"Does anyone know X?", "Can someone help me with Y?", or "Are there any experts in Z?" '
+        "just ask your question or describe your problem directly."
+    ),
```
Comment (GabeMillikan) [inline cogs/Tips.py:17]: replaces `/ask`
image

File: cogs/Tips.py [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+        "This message:\n"
+        "**\\`\\`\\`java\n"
+        'System.out.println("Code formatting is neat!");\n'
+        "\\`\\`\\`**\n\n"
+        "Will look like this:\n"
+        "```java\n"
+        'System.out.println("Code formatting is neat!");\n'
+        "```\n"
+        "Go ahead, try it! You can copy/paste the message above."
+    ),
```
Comment (GabeMillikan) [inline cogs/Tips.py:32]: Replaces `/codehelp`
image

File: cogs/Tips.py [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+        "This way, we can start helping you right where you left off and can save everyone's time."
+    ),
+    "Use the Mention System": (
+        "If some time has passed and nobody is responding to your question, then you can use "
+        " the `/mention create [role]` command to mention everyone with the relevant role, such as @Math or @Biology.\n\n"
+        "You can select a certain message to mention for by right clicking on a message and selecting Apps -> mention.\n\n"
+        "After waiting 10 minutes, you can use the `/mention send` which will ping the role with your message.\n\n"
+        "If you decide to cancel your request, you can use `/mention cancel`, or if you want to create a different "
+        "mention you can use `/mention overwrite [role]` which works similarly to `/mention create [role]`."
+    ),
```
Comment (GabeMillikan) [inline cogs/Tips.py:41]: From #tips
image

File: cogs/Tips.py [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+    "Try Googling It": (
+        "There are many resources available online, so googling your problem should be the first course of action.\n"
+        "Very frequently, you can get an answer more quickly and with more detail.\n"
+        "If you've already tried googling the problem, then let us know what you found and why it wasn't sufficient!"
+    ),
```
Comment (GabeMillikan) [inline cogs/Tips.py:46]: From #tips
image

**Review by Arborym — APPROVED** [review]

**Review by sebastiaan-daniels — APPROVED** [review]

**Pull request merged**